linetools/spectra/io.py

```diff
@@ -199,7 +202,11 @@ def readspec(specfil, inflg=None, efil=None, verbose=False, multi_ivar=False,
 
     # Add in the header
     #xspec1d.head = head0
-    xspec1d.meta['headers'][0] = head0
+    if head_ext == 0:
```

Comment:
Got it; then I suggest to store `head0` and e.g. `head_spec` variables both in the same part of the code. Please also add inline comments pointing out that `head0` contains info that `head_spec` may not necessarily have.